frontend_app_admin_workspace_user/dist/index.html

```diff
@@ -55,20 +104,32 @@
 
   <script type='text/javascript'>
     GLOBAL_primaryColor = '#7d4e24'
-    GLOBAL_dispatchEvent = function (e) { console.log('GLOBAL_dispatchEvent', e) }
+    GLOBAL_dispatchEvent = () => { }
+    tinymce = {
+      remove: () => { }
+    }
+    wysiwyg = () => { }
   </script>
 </head>
+
 <body>
   <script src="./assets/bootstrap/jquery-3.2.1.js"></script>
   <script src="./assets/bootstrap/popper-1.12.3.js"></script>
   <script src="./assets/bootstrap/bootstrap-4.0.0-beta.2.js"></script>
 
   <div id='content'></div>
 
-  <script type='text/javascript' src='./app/tracim_frontend_vendors.js'></script>
-  <script type='text/javascript' src='./app/tracim_frontend_lib.lib.js'></script>
+  <!--  <script type='text/javascript' src='./app/tracim_frontend_vendors.js'></script>-->
+  <!--  <script type='text/javascript' src='./app/tracim_frontend_lib.lib.js'></script>-->
+
+  <!--  <script type='text/javascript' src='../frontend/src/util/appInterface.js'></script>-->
+  <!--  <script type='text/javascript' src='../frontend/src/util/tinymceInit.js'></script>-->
```

Comment:
Are these comments intentional or forgotten?

Reply:
mmh intentional.
To mark the fact that we deliberately don't use theses scripts in standalone mode.

But I should add some comment for explaination

Reply:
fixed